geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/process/tube/TubeSelectProcessTest.scala

```diff
@@ -259,8 +262,141 @@ class TubeSelectProcessTest extends Specification {
 
       results.size mustEqual 20
     }
+
+    "should properly handle geometries crossing the IDL" in {
```

Comment:
`<curmudgeon_mode>`I know that nobody cares, but this is NOT a fix for the IDL (a complicated line segment); it is a fix for the anti-meridian.`</curmudgeon_mode>`

Reply:
plus, 'anti-meridian' is a cool word

Reply:
we need tshirts

Reply:
So should I change all instances of IDL to read AntiMeridian or AM for accuracy's sake?